VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimStatus.test.tsx

```diff
@@ -51,44 +41,48 @@ context('ClaimStatus', () => {
   })
 
   it('should initialize', async () => {
-    expect(component).toBeTruthy()
+    expect(screen.getAllByText('You have 2 file requests from VA')).toBeTruthy()
+    expect(screen.getByTestId('Step 1 of 5. completed. Claim received June 6, 2019')).toBeTruthy()
+    expect(screen.getByTestId('Step 2 of 5. completed. Initial review June 6, 2019')).toBeTruthy()
+    expect(screen.getByTestId('Step 3 of 5. current. Evidence gathering, review, and decision July 16, 2020')).toBeTruthy()
+    expect(screen.getByTestId('Step 4 of 5.  Preparation for notification')).toBeTruthy()
+    expect(screen.getByTestId('Step 5 of 5.  Complete')).toBeTruthy()
+    expect(screen.getByText('Why does VA sometimes combine claims?')).toBeTruthy()
+    expect(screen.getByText("What should I do if I disagree with VA's decision on my disability claim?")).toBeTruthy()
+    expect(screen.getByText('Need help?')).toBeTruthy()
+    expect(screen.getByText('Call our VA benefits hotline. We’re here Monday through Friday, 8:00 a.m. to 9:00 p.m. ET.')).toBeTruthy()
+    expect(screen.getByText('800-827-1000')).toBeTruthy()
   })
 
   describe('when the claimType is ACTIVE', () => {
     describe('on click of Find out why we sometimes combine claims. list item', () => {
       it('should call useRouteNavigation', async () => {
-        testInstance.findAllByType(Pressable)[4].props.onPress()
+        fireEvent.press(screen.getByText('Why does VA sometimes combine claims?'))
         expect(mockNavigationSpy).toHaveBeenCalledWith('ConsolidatedClaimsNote')
         expect(mockNavigationResultSpy).toHaveBeenCalledWith()
       })
     })
 
     describe('on click of What should I do if I disagree with VA’s decision on my disability claim? list item', () => {
       it('should call useRouteNavigation', async () => {
-        testInstance.findAllByType(Pressable)[5].props.onPress()
+        fireEvent.press(screen.getByText("What should I do if I disagree with VA's decision on my disability claim?"))
```

Comment:
This can also be queried with `getByRole`
```tsx
fireEvent.press(screen.getByRole('button', { name: "What should I do if I disagree with VA's decision on my disability claim?" }))
```

Reply:
changed